.jscsrc

```diff
@@ -46,7 +46,7 @@ requirePaddingNewLinesAfterBlocks: { allExcept: ['inCallExpressions', 'inNewExpr
 requirePaddingNewLinesAfterUseStrict: true
 requirePaddingNewLinesBeforeExport: true
 requirePaddingNewLinesBeforeLineComments: { allExcept: firstAfterCurly }
-requirePaddingNewlinesBeforeKeywords: [ case, do, finally, for, function, if, return, switch, try, typeof, void, while, with ]
+requirePaddingNewlinesBeforeKeywords: [ case, do, finally, for, if, return, switch, try, typeof, void, while, with ]
```

Comment:
We are using this format for other rules also. Either we keep it like this or we change them all. In any case, this is a perfectly valid format according to the latest YAML [spec](http://www.yaml.org/spec/1.2/spec.html).